mpa/det/inferrer.py

```diff
@@ -113,18 +113,20 @@ def infer(self, cfg, eval=False, model=None, dump_features=False, dump_saliency_
             target_classes = dataset.CLASSES
 
         # Model
-        if model is None:
-            cfg.model.pretrained = None
-            if cfg.model.get('neck'):
-                if isinstance(cfg.model.neck, list):
-                    for neck_cfg in cfg.model.neck:
-                        if neck_cfg.get('rfp_backbone'):
-                            if neck_cfg.rfp_backbone.get('pretrained'):
-                                neck_cfg.rfp_backbone.pretrained = None
-                elif cfg.model.neck.get('rfp_backbone'):
-                    if cfg.model.neck.rfp_backbone.get('pretrained'):
-                        cfg.model.neck.rfp_backbone.pretrained = None
-
+        cfg.model.pretrained = None
+        if cfg.model.get('neck'):
+            if isinstance(cfg.model.neck, list):
+                for neck_cfg in cfg.model.neck:
+                    if neck_cfg.get('rfp_backbone'):
+                        if neck_cfg.rfp_backbone.get('pretrained'):
+                            neck_cfg.rfp_backbone.pretrained = None
+            elif cfg.model.neck.get('rfp_backbone'):
+                if cfg.model.neck.rfp_backbone.get('pretrained'):
+                    cfg.model.neck.rfp_backbone.pretrained = None
```

Comment:
when model doesn't contain neck, `cfg.model.pretrained` is going to be `None`. Is this okay?